server/models/Order.ts

```diff
@@ -20,12 +20,12 @@ const OrderModel = z.object({
   id: z.string().uuid(),
   status: OrderStatusEnum,
   deviceWearer: DeviceWearerModel,
-  addresses: z.array(AddressModel),
+  addresses: z.array(AddressModel).nullish(),
```

Comment:
Why has this been made nullable? Its always a list

Reply:
not when I was running it... maybe that was before fixes got applied?